editor/src/templates/editor.tsx

```diff
@@ -325,9 +326,13 @@ export class Editor {
     )
 
     void getLoginState('cache').then((loginState: LoginState) => {
+      const projectId = getProjectID()
       startPollingLoginState(this.boundDispatch, loginState)
       this.storedState.userState.loginState = loginState
-      void getUserConfiguration(loginState).then((shortcutConfiguration) => {
+      void Promise.all([
```

Comment:
see the comment about checking the BFF enabled in `getUserPermissions` or this whole promise will throw